application/classes/Ushahidi/Repository/User.php

```diff
@@ -64,7 +64,11 @@ public function create(Entity $entity)
 			'created'  => time(),
 			'password' => $this->hasher->hash($entity->password),
 		];
-		return parent::create($entity->setState($state));
+		$entity->setState($state);
+		if ($entity->role === 'admin') {
```

Comment:
Ideally this check happens in the listener too. Then the event is reusable for future integrations that might want *all* users.